build.xml

```diff
@@ -45,15 +45,15 @@
     <target name="clean">
         <delete failonerror="false" includeEmptyDirs="true">
             <fileset dir="${output}"/>
-            <fileset file="*.jar"/>
+            <fileset file="${basedir}/*.jar"/>
```

Comment:
jar files may not be placed in ${basedir}. because cubrid-jar-file and cubrid-src-jar-file properties are exposed.
Also, bin-cubrid and src-cubrid are not guaranteed in the output folder

Reply:
suggestion:
```
            <fileset dir="${bin-cubrid}"/>
            <fileset dir="${src-cubrid}"/>
            <fileset dir="${output}"/>
            <fileset file="${cubrid-jar-file}"/>
            <fileset file="${cubrid-src-jar-file}"/>
            <fileset file="*.jar"/>
```